packages/core/addon/mixins/perfect-scrollbar.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import Mixin from '@ember/object/mixin';
+import computed from '@ember/object/computed';
+import { get, set } from '@ember/object';
+import { isPresent } from '@ember/utils';
+
+const { PerfectScrollbar } = window;
+
+export default Mixin.create({
```

Comment:
Do we need this mixin? Are we expecting to use this without the component?

Reply:
The mixin was grafted from elsewhere.  Since it was already a mixin, i decided to keep it that way.  It could be useful if a component needs to become scrollable.